underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -554,9 +554,11 @@ private ObjectPermissions getPermissionsInternal() {
 
         bucketMode = S3AUtils.translateBucketAcl(acl, owner.getId());
         if (mUfsConf.isSet(PropertyKey.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING)) {
+          // Here accountOwner can be null if there is no mapping set for this owner id
           accountOwner = CommonUtils.getValueFromStaticMapping(
               mUfsConf.get(PropertyKey.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING), owner.getId());
-        } else {
+        }
+        if (accountOwner == null || accountOwner.equals(DEFAULT_OWNER)) {
```

Comment:
can you also update the `PropertyKey` description for `UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING` to reflect this behavior